package.json

```diff
@@ -24,26 +24,22 @@
   ],
   "scripts": {
     "build": "tsc",
+    "check": "tsc --noEmit",
+    "clean": "git clean -fXd",
```

Comment:
Could you put this change into a separate commit, since it's not directory related to removing uri.js?

Reply:
Done